README.md

```diff
@@ -95,26 +94,30 @@ history = reenact_events(stoichiometry, result['events'], state)
 
 There are more command line features in test_arrow:
 
-    > python -m arrow.test.test_arrow --complexation
+    > python -m stochastic_arrow.test.test_arrow --complexation
 
-    > python -m arrow.test.test_arrow --plot
+    > python -m stochastic_arrow.test.test_arrow --plot
 
-    > python -m arrow.test.test_arrow --obsidian
+    > python -m stochastic_arrow.test.test_arrow --obsidian
 
-    > python -m arrow.test.test_arrow --memory
+    > python -m stochastic_arrow.test.test_arrow --memory
 
-    > python -m arrow.test.test_arrow --time
+    > python -m stochastic_arrow.test.test_arrow --time
 
 More examples:
 
-    > python -m arrow.test.test_hang
+    > python -m stochastic_arrow.test.test_hang
 
-    > pytest -m arrow/test/test_arrow.py
+    > pytest -m stochastic_arrow/test/test_arrow.py
 
     > pytest -k flagella
 
 ## Changelog
 
+### Version 0.5.3
+
+* Rename module to `stochastic_arrow` to avoid name conflict (Issue #51).
```

Comment:
* A totally breaking change to clients calls for a major version number change. So let's go to `1.0.0`.
* Let's make this change note bolder: Every project needs to update their import statements, changing `arrow` to `stochastic_arrow`.

* Let's remember to test that `pip install -U` from arrow 0.5.2 to this new release (1) successfully removes the `arrow/` directory from site-packages in the case where the `arrow` date-time package was not installed, and (2) doesn't remove it in the case where the `arrow` date-time package installed its code into the `arrow/` directory. -- If this test fails, maybe there's a workaround in `setup.py`. Otherwise we'll have to give manual workaround instructions to people.

Reply:
Good idea. While upgrading directly to this release does handle (1), it fails (2). I added a note to the README telling people to uninstall `arrow` before upgrading this package, then reinstall it. I tried automating this in `setup.py` by using `subprocess` to call `pip` but could not get it to work when installing from a source distribution. Plus it seemed like a very bad idea anyways.